src/edges.jl

```diff
@@ -0,0 +1,38 @@
+# Create recipe: 
+
+@Makie.recipe(Edges, grid) do scene
+    Makie.Attributes(;
+
+      # Use default theme:
+      Makie.default_theme(scene, Makie.Lines)...,
+
+      # Custom arguments:
+      color      = :brown4,
+      colormap   = :bluesreds,
+      colorrange = nothing
+    )
+end
+  
+  # Plot! especialization:
+  
+function Makie.plot!(plot::Edges{<:Tuple{Grid}})
+  
+  # Retrieve plot arguments:
+  grid      = plot[:grid][]
+  color     = plot[:color][]
+
+  if color isa AbstractVector
```

Comment:
Why does this recipe needs to handle `color` ? This is already done in `faces` right?